ext/drwrap/drwrap.c

```diff
@@ -974,6 +974,17 @@ drwrap_exit(void)
     if (count != 0)
         return;
 
+    ASSERT(drmgr_unregister_bb_app2app_event(drwrap_event_bb_app2app),
```

Comment:
Do the call outside the ASSERT: as is, this call will disappear in release build.  Ditto below.